Tests/Foundation/TestFileManager.swift

```diff
@@ -1385,15 +1393,11 @@ class TestFileManager : XCTestCase {
 
         XCTAssertThrowsError(try fm.attributesOfItem(atPath: "")) {
             let code = ($0 as? CocoaError)?.code
-            XCTAssertEqual(code, .fileReadInvalidFileName)
+            XCTAssertEqual(code, .fileReadNoSuchFile)
```

Comment:
Do we simply throw a different error now, or is the former API somehow missing?

Reply:
We throw an error now. Previously the behavior for these empty paths differed between Darwin and non-Darwin. The swift-foundation implementation takes the behavior from Darwin and so this updates the test expectations accordingly. Previously, getting the file system representation of an empty string unconditionally threw `.fileReadInvalidFileName`, but now we just create en empty cstring and provide it to the underlying system API and let it determine how to handle an empty path (which in most cases results in specifying that the file doesn't exist, unless you're working with a directory in which I believe some cases it can be considered the current directory)